README.md

```diff
@@ -182,7 +131,8 @@ This SDK abides by our Client SDK Deprecation Policy. For more information on th
 
 | Major version number | Status | Released | Deprecated | Unsupported |
 | -------------------- | ------ | -------- | ---------- | ----------- |
-| 2.x.x | Active | October 2023 | TBA | TBA |
+| 3.x.x | Active | April 2023 | TBA | TBA |
+| 2.x.x | Active | October 2023 | April 2025 | TBA |
```

Comment:
We will want to either leave 3.x release date as TBA or update to March 2025. For 2.x the unsupported date would be 1 year after deprecated and deprecated would be as soon as 3.x is released. So I think these should be:
```suggestion
| 3.x.x | Active | March 2025 | TBA | TBA |
| 2.x.x | Deprecated | October 2023 | March 2025 | March 2026 |
```
@sarahkoop to keep me honest